src/styles/components/tile.scss

```diff
@@ -31,12 +35,32 @@ $tile: '.tile';
     margin: 0;
     line-height: 1.2;
   }
-  padding: $global-padding/2 0 0 0;
-  cursor: pointer;
 
   &__header {
+    @include fs-font-settings('H3');
     display: block;
+    margin-top: 0.5em;
+
+    // match mixin specificity...
+    @media (min-width: 0px) {
```

Comment:
What does this do?

Reply:
It just matched the specificity of `fs-font-settings` when I was using it, but with the new helper classes it's not needed anymore